spec.html

```diff
@@ -13626,7 +13643,7 @@ <h2>Syntax</h2>
 
       <emu-clause id="sec-static-semantics-mv">
         <h1>Static Semantics: MV</h1>
-        <p>A numeric literal stands for a value of the Number type or the BigInt type.</p>
+        <p>A numeric literal stands for a value of the Number type or the BigInt type, as specified by the syntax-directed operation NumericValue. Determining this value involves deriving a mathematical value (MV) from the literal, according to the following rules.</p>
```

Comment:
```suggestion
        <p>A numeric literal stands for a value of the Number type or the BigInt type, as specified by the syntax-directed operation NumericValue. Determining this value involves deriving a mathematical value from the literal, according to the following rules.</p>
```

MV is just an SDO name here.

Reply:
I agree it's an SDO name, but I don't get how that explains the removal.

I imagine it's there to explain where the name "MV" comes from.

Reply:
I don't think this is a clear enough way to explain the initialism. If that's the goal, you should say something like "the origin for the name MV". But I don't really think it's necessary either.

Reply:
> I don't think this is a clear enough way to explain the initialism.

It seems like the rest of the spec disagrees. E.g.:
- "mathematical value (MV)" (3 occurrences in status quo)
- "String value (SV)"
- "Template Value (TV)"
- "Template Raw Value (TRV)"
- "application programming interface (API)"
- "exclusive OR (XOR)"
- "Basic Multilingual Plane (BMP)"
- "LINE FEED (LF)"
- "depth-first search (DFS)"

> If that's the goal, you should say something like "the origin for the name MV".
> But I don't really think it's necessary either.

I agree that it's not necessary to say "The origin for the name MV is that it's an acronym for 'mathematical value'."

But I'll remove "(MV)" here if another editor thinks that doing so is an improvement.

Reply:
@jmdyck There's a difference between "application programming interface (API)" and "mathematical value (MV)". In the first case, they are replaceable: you can use either in any context. For "MV", we only use it to refer to the AO. We never say "the MV of" as short-hand for "the mathematical value of".

Reply:
Okay, so are you saying that that difference makes "application programming interface (API)" a clear way to explain "API", but "mathematical value (MV)" a not-clear-enough way to explain "MV"?

Reply:
Yeah because they're using the same presentation to do two different things.